pytorch_lightning/trainer/trainer.py

```diff
@@ -415,22 +414,22 @@ def __init__(
     def fit(
         self,
         model: LightningModule,
-        train_dataloader: Any = None,
-        val_dataloaders: Optional[Union[DataLoader, List[DataLoader]]] = None,
+        train_dataloaders: Optional[Union[TRAIN_DATALOADERS, LightningDataModule]] = None,
```

Comment:
Not sure about `train_dataloaders` as we support only 1 train_dataloader object. Mind explaining ?

Reply:
Also, not a huge fan of assigning train_dataloaders to LightingDataModule.